_data/osdf_namespace_metadata/pelicanplatform.yaml

```diff
@@ -1,11 +1,12 @@
 description: |
+  Testing and Validation Origin
 
 organization: # Name of the organization
-dataVisibility: # public, or private
-size: # In bytes so I can parse how I want
+dataVisibility: public # public, or private
+size: 8132  # In bytes so I can parse how I want
 bytesXferd: # In bytes so I can parse how I want
-url: # Organization URL
-fieldOfScience: # Top level Field of Science pulled from here https://nces.ed.gov/ipeds/cipcode/browse.aspx?y=55
+url: https://pelicanplatform.org  # Organization URL
+fieldOfScience: COMPUTER AND INFORMATION SCIENCES AND SUPPORT SERVICES  # Top level Field of Science pulled from here https://nces.ed.gov/ipeds/cipcode/browse.aspx?y=55
```

Comment:
We've settled on regular title casing for these -- can you change to "Computer and Information Sciences..."?

Reply:
D'oh, forgot to push. Thanks